gammapy/datasets/map.py

```diff
@@ -818,6 +866,13 @@ def plot_residuals_spatial(
         -------
         ax : `~astropy.visualization.wcsaxes.WCSAxes`
             WCSAxes object.
+
+        Example
+        -------
+        >>> from gammapy.datasets import MapDataset
+        >>> dataset = MapDataset.read("$GAMMAPY_DATA/cta-1dc-gc/cta-1dc-gc.fits.gz")
+        >>> kwargs = {"cmap": "gnuplot2", "add_cbar": True}
```

Comment:
For residual plotting I'd always use `cmap="RdBu"`  and a symmetric data range, like `vmin=-5, vmax=5`

Reply:
I always use `"RdBu_r"` since most people associate blue with negative and red with positive changes (colder / warmer)